src/views/administration/AdminMenu.vue

```diff
@@ -24,7 +24,7 @@
       >
         <div class="list-group" id="list-tab" role="tablist">
           <router-link
-            :ref="item.id"
+            :key="item.id"
```

Comment:
My experience with Vue is limited, but based on the docs it doesn't seem like [`ref`](https://v2.vuejs.org/v2/api/#ref) and [`key`](https://v2.vuejs.org/v2/api/#key) are the same thing, semantically.

I can't really tell if `ref` was ever needed here, but on first look it seems wrong to simply replace it with `key`, instead of simply adding `key`. Any thoughts?